include/vapor/DCBOV.h

```diff
@@ -16,6 +16,33 @@ namespace VAPoR {
 
 class BOVCollection;
 
+// Example BOV header file, bovA1.bov
+//
+// # TIME is a floating point value specifying the timestep being read in DATA_FILE
+// TIME: 1.1
+//
+// # DATA_FILE points to a binary data file.  It can be a full file path, or a path relative to the BOV header.
+// DATA_FILE: bovA1.bin
+//
+// # The data size corresponds to NX,NY,NZ in the above example code.  It must contain three values
+// DATA_SIZE: 10 10 10
+//
+// # Allowable values for DATA_FORMAT are: INT,FLOAT,DOUBLE
+// DATA_FORMAT: FLOAT
+//
+// # VARIABLE is a string that specifies the variable being read in DATA_FILE.  Must be alphanumeric (abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ01234567890)
```

Comment:
I would suggest adding a hyphen and an underscore into the allowed list, since those 2 are pretty common and useful in variable names.

Reply:
No problem.  I'll push soon.

Reply:
could you also remove the duplicate `0` (zero) in this allowed list? It helps the code to look better.

Reply:
Sure, fixed.  Good eye.